align/cell_fabric/drc.py

```diff
@@ -51,8 +51,12 @@ def _check_via_enclosure_rules(self, layer, vv):
         def check_single_metal( r, ly, metal_dir, enclosure_value):
             o = 1 if metal_dir == 'V' else 0
             metal_r = self._find_rect_covering_via( r, ly, metal_dir)
-            if metal_r[o+0] > r.rect[o+0] - enclosure_value or metal_r[o+2] < r.rect[o+2] + enclosure_value:
-                self.errors.append( f"Enclosure violation on {ly}-{layer}: {metal_r} does not sufficiently surround {r.rect}, {enclosure_value}")
+            # try statement added by kunal to skip failure for bulk pdk
+            try:
+                if metal_r[o+0] > r.rect[o+0] - enclosure_value or metal_r[o+2] < r.rect[o+2] + enclosure_value:
```

Comment:
What was the failure for bulk pdk that you are trying to mask?

Reply:
While running Nightly regressions through bulk PDK I found that I was not able to generate Resistors due to this check. I think the bulk JSON has some issues, so I skipped this check for now. I can create a separate issue to come back and see this again later.

Reply:
Please create an issue with a failing circuit and assign @arvuce22 and @parijatm to it.